package.json

```diff
@@ -111,6 +113,7 @@
     "moment": "^2.25.3",
     "multistream": "^4.0.0",
     "node-fetch": "^2.6.0",
+    "p-queue": "^6.6.1",
```

Comment:
Lots of these are for the build process I'm creating for the vector tiles. Sorry for letting them leak in here